src/appshell/internal/applicationactioncontroller.cpp

```diff
@@ -41,6 +41,11 @@ using namespace mu::actions;
 void ApplicationActionController::preInit()
 {
     qApp->installEventFilter(this);
+
+    //! NOTE: process QEvent::FileOpen as early as possible if it was postponed
+#ifdef Q_OS_MACOS
+    qApp->processEvents();
+#endif
```

Comment:
I think it's better to call `processEvents` in `onAllInited` because in `preInit` modules are not initialized

Reply:
Yes, let's try